src/content/news/2025-05-04-maplibre-newsletter-april-2025/index.mdx

```diff
@@ -0,0 +1,138 @@
+
+- **Plugin Support for Custom Layers (WIP)**: There is a [draft PR](https://github.com/maplibre/maplibre-native/pull/3430) that adds plugin functionality to MapLibre Native, allowing **custom layer types** to be added at runtime. Allowing the library to be extended more easily from the outside is something that has been talked about a long time. Although the PR in question is very much a work-in-progress, it is exciting to see some movement in this area which many users will be following with interest.
+
+- **Call for Accessibility Feedback**: We are interested in input from **accessibility** experts or developers who developed accessible maps in the past. Do you have any ideas for MapLibre Native features or perhaps style spec additions that can benefit accessibility? If yes, then please [add to the discussion](https://github.com/maplibre/maplibre-native/discussions/3439).
+
+- Releases this month:
+  - MapLibre iOS [6.12.2](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.12.2), [6.12.3](https://github.com/maplibre/maplibre-native/releases/tag/ios-v6.12.3)
+  - MapLibre Android [11.8.6](https://github.com/maplibre/maplibre-native/releases/tag/android-v11.8.6)
+  - MapLibre Node.js [6.1.0](https://github.com/maplibre/maplibre-native/releases/tag/node-v6.1.0).
+
+## Web
+
+We released two versions this month: - [v5.3.1](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.3.1): A minor update focused on bug fixes. - [v5.4.0](https://github.com/maplibre/maplibre-gl-js/releases/tag/v5.4.0): A major release introducing Level of Detail (LOD) controls based on pitch and zoom levels, giving developers more flexibility in how data is rendered dynamically.
+
+Looking ahead, map-state persistence and expanded hillshade options are in the pipeline, stay tuned! There’s also early progress on a plugin to support complicated scripting languages, a feature that’s been discussed in previous community posts.
+
+Be sure to explore the [awesome-maplibre](https://github.com/maplibre/awesome-maplibre) repository. It’s packed with creative tools, plugins, and use cases built by our amazing community!
+
+## Martin Tile Server
+
+We’re actively shaping the roadmap for Martin v1.0 and would love your input!
+
+A discussion is underway on what features or changes are needed before we consider a 1.0 release. If you have thoughts, suggestions, or feature requests, [please join the conversation](https://github.com/maplibre/martin/discussions/1805) and help guide the future of Martin.
+
+## 🌟 Community Spotlight
+
+**[GTFS Visualization for Public Transport in Zamora, Mexico](https://rutaton.zamora.gob.mx)**
```

Comment:
This is not open source, but it is not blatant product promotion either so I think it's OK to include.

Maybe we should define somewhere what types of submissions we want for the newsletter (maybe you can make a new issue to gather feedback for the newsletter).